consent-ws/src/main/java/org/broadinstitute/consent/http/service/DataAccessParser.java

```diff
@@ -194,6 +195,43 @@ public PDAcroForm fillDARForm(Document dar, Map<String, String> researcherProper
                     field.setValue(getYesOrNoValue(dar.getBoolean(DarConstants.POA)));
                     break;
                 }
+                case DarConstants.CHECK_COLLABORATOR: {
+                    field.setValue(getYesOrNoValue(dar.getBoolean(DarConstants.CHECK_COLLABORATOR)));
+                    break;
+                }
+                case "userStatus": {
```

Comment:
Any reason these case strings (here and below) are not a constant like we have for other cases?